tmc-langs-r/src/main/java/fi/helsinki/cs/tmc/langs/r/RPlugin.java

```diff
@@ -0,0 +1,158 @@
+package fi.helsinki.cs.tmc.langs.r;
+
+
+import fi.helsinki.cs.tmc.langs.AbstractLanguagePlugin;
+import fi.helsinki.cs.tmc.langs.abstraction.ValidationResult;
+import fi.helsinki.cs.tmc.langs.domain.ExerciseBuilder;
+import fi.helsinki.cs.tmc.langs.domain.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.domain.RunResult;
+import fi.helsinki.cs.tmc.langs.domain.TestDesc;
+import fi.helsinki.cs.tmc.langs.io.StudentFilePolicy;
+import fi.helsinki.cs.tmc.langs.io.sandbox.StudentFileAwareSubmissionProcessor;
+import fi.helsinki.cs.tmc.langs.io.zip.StudentFileAwareUnzipper;
+import fi.helsinki.cs.tmc.langs.io.zip.StudentFileAwareZipper;
+
+import fi.helsinki.cs.tmc.langs.utils.ProcessRunner;
+
+
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableList;
+
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Locale;
+
+public final class RPlugin extends AbstractLanguagePlugin {
+
+    /*
+    R folder contains the actual R files used in the
+    project/package. It is automatically included when creating a
+    R package but now when making a regular project in RStudio.
+     */
```

Comment:
Maybe all these comments should follow the Javadoc convention that is used throughout.

For example this comment should look like this:
```java
    /**
     * R folder contains the actual R files used in the
     * project/package. It is automatically included when creating a
     * R package but now when making a regular project in RStudio.
     */
```

I'm not going to mention this in other places but please go through your code and fix it accordingly.